assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -343,4 +344,98 @@ describe('ExecutionResults', () => {
     expect(screen.getAllByText('2/2 expectations passed')).toBeTruthy();
     expect(screen.getAllByText('1/2 expectations failed')).toBeTruthy();
   });
+
+  it('given provider is VMware, should render ExecutionResults with warning banner', async () => {
+    const {
+      clusterID,
+      hostnames,
+      checks: [checkID1, checkID2],
+      loading,
+      catalog,
+      error,
+      executionLoading,
+      executionData,
+      executionError,
+      executionStarted,
+    } = prepareStateData('passing');
+
+    renderWithRouter(
+      <ExecutionResults
+        clusterID={clusterID}
+        clusterName="test-cluster"
+        clusterScenario="hana_scale_up"
+        cloudProvider="vmware"
+        hostnames={hostnames}
+        catalogLoading={loading}
+        catalog={catalog}
+        executionStarted={executionStarted}
+        catalogError={error}
+        executionLoading={executionLoading}
+        executionData={executionData}
+        executionError={executionError}
+      />,
+      { route: `/clusters/${clusterID}/executions/last?health=passing` }
+    );
+
+    expect(screen.getByText('test-cluster')).toBeTruthy();
```

Comment:
I think most of this `expect`a are redundant. Wouldn't be enough with the `VMware` and the warning message expects. To not have so many tests checking the same.
The same goes for the next test.